metrics/metrics/src/main/java/module-info.java

```diff
@@ -32,4 +34,6 @@
     requires microprofile.config.api;
 
     exports io.helidon.metrics;
+
+    uses ExemplarService;
```

Comment:
please use fully qualified class name - it is approach we use in most modules, so please keep it consistent

Reply:
Fixed